src/vendor/vendor.c

```diff
@@ -1704,11 +1693,7 @@ int RdkResetInner(char *group_name, char *path, char *value, char *debug_msg)
 **************************************************************************/
 int DiscoverDM_Components(char *comps_filename)
```

Comment:
The old CCSP APIs needed to know which software component owned each object, but this is not the case for RBus APIs, so there is no need to discover this, nor to have to put this info in the usp pa config files (usp_dm_comps.conf, usp_dm_objs.conf and usp_dm_params.conf).
The usp_dm_comps.conf can be removed entirely (along with the code that generates and parses it) and a single group_id in OBUSPA can be used for the whole of the RBus data model. This is more efficient and also more conformant with USP (eg R-Bus Sets can be atomic across more than one component, which wasn't possible with CCSP API).

Reply:
Addressed by Removed the usp_dm_comps.conf, defined GROUP_Id as 1
Please review.